SPEC.md

```diff
@@ -2274,9 +2414,23 @@ Default unit is Bytes ("B").
 
 ## String sub(String, String, String)
 
+:pig2: [Cromwell supported](https://github.com/broadinstitute/cromwell#wdl-support) :white_check_mark:
+
 Given 3 String parameters `input`, `pattern`, `replace`, this function will replace any occurrence matching `pattern` in `input` by `replace`.
 `pattern` is expected to be a [regular expression](https://en.wikipedia.org/wiki/Regular_expression). Details of regex evaluation will depend on the execution engine running the WDL.
 
+## Array[Int] range(Int)
```

Comment:
@geoffjentry This section wasn't in the spreadsheet, is it supported by Cromwell? I'm not sure why it shows that I added this section, Array[Int] range(Int) and the next few, I think it was a Github error. Does it belong in the spec? 
@ruchim did you add these sections? I might have adopted your changes and now it looks like I made them. Are they supported by Cromwell?